WeslToml.md

```diff
@@ -0,0 +1,87 @@
+# `wesl.toml` file format
+
+WESL shader projects have a standardized `wesl.toml` file, which describes the project setup. It is used by language servers and linkers[^1].
+
+The format looks as follows. As much of it is optional as possible.
+
+```toml
+[package] # Following the toml convention of grouping properties.
+
+# Optional, used for specifying the name within wesl shaders.
+# After all, the package manager's naming convention might clash with the WGSL naming conventions.
+# (E.g. camelCase vs snek_case)
+# Also deals with names like `@angular/animations`. Solves https://github.com/wgsl-tooling-wg/wesl-spec/issues/76
+name = "angular_animations" 
+
+# Not optional. Rust initially made it optional and it was a footgun.
+# It ended up being too easy to opt into the semantics of the oldest Rust edition.
+edition = "2025"
+
+# Description and version fields do not exist.
+# They match the package.json/Cargo.toml.
+# Semantics are "we are publishing a normal package that happens to also contain wesl code".
+# This lets us support wesl packages that come with host code!
+
+# Where are the shaders located. This is the path of `package::`.
+# We watch this directory for changes.
+root = "./shaders"
+
+# Some projects have large folders that we shouldn't react to. Globs for excluding them.
+exclude = [ "**/test" ]
+
+# Can be auto-inferred from the existence of a package.json.
+# Necessary when both a `package.json` and a `Cargo.toml` are present.
+# Inclusion of this field is gently encouraged.
+package-manager = "npm"
+
+# Optional dependencies section, automatically obtained from the package manager.
+[dependencies]
+ # Has no effect.
+foolib = {}
+ # Can be used for renaming packages. Now bevy in my code is called "cute_bevy".
+cute_bevy = { package = "bevy" }
```

Comment:
Make it clear what names go here when a package has a different name for wesl.

(Left side: wesl name. Right side: Original name.)